src/styles/blocks/header.scss

```diff
@@ -0,0 +1,52 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+  }
+
+  &__ul-list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    margin-left: 20px;
+    padding: 23px 0;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-weight: 500;
+    color: #{$main-text-color};
+
+    &:first-child {
+      margin: 0;
+    }
+
+    &:hover {
+      color: #{$button-color};
+    }
+
+    &.is-active {
+      color: #{$button-color};
+    }
+  }
+}
+
+.is-active:after {
+  position: absolute;
+  content: "";
+  height: 4px;
+  width: 100%;
+  bottom: 0;
+  background-color: #{$button-color};
```

Comment:
don't need tipe #{} when connect variables in one file through import, use only $ (everywhere)